### ISSUE TITLE
Regression: osc chroot does not accept `--force` option with --wipe.

### ISSUE THREAD

**Issue by nickbroon** [openSUSE/osc#706]
```
osc chroot --wipe --force --root /var/lib/jenkins/workspace/job/build-root/project-standard-x86_64
osc chroot: no such option: --force
Try 'osc help chroot' for info.
```

man page states:

```
                  -f, --force
                              Do not ask confirmation for wipe
                  --wipe      Delete the build root instead of chrooting into it
```

**Comment by adrianschroeter** [comment]
On Dienstag, 10. Dezember 2019, 15:21:41 CET Nick Brown wrote:
> ```
> osc chroot --wipe --force --root /var/lib/jenkins/workspace/job/build-root/project-standard-x86_64
> osc chroot: no such option: --force
> Try 'osc help chroot' for info.
> ```
> 
> man page states:
> 
> ```
>                   -f, --force
>                               Do not ask confirmation for wipe
>                   --wipe      Delete the build root instead of chrooting into it
> ```

I think we should just drop the -f part from the help ... can do so


-- 

Adrian Schroeter <adrian@suse.de>
Build Infrastructure Project Manager

SUSE Software Solutions Germany GmbH,  Maxfeldstr. 5, 90409 Nuernberg, Germany
(HRB 247165, AG München), Geschäftsführer: Felix Imendörffer

**Comment by nickbroon** [comment]
This used to work.
Without the `--force` option how would a non-interactive script remove the build chroot?

```
~$ osc chroot --wipe --force --root=/var/lib/jenkins/workspace/job/build-root/project-standard-x86_64
osc chroot: no such option: --force
Try 'osc help chroot' for info.

~$ osc chroot --wipe --root=/var/lib/jenkins/workspace/job/build-root/project-standard-x86_64
Error: '/home/user' is not an osc project dir or working copy
```

**Comment by lethliel** [comment]
@nickbroon @adrianschroeter 

This is a regression. Wiping the local build root should be possible without the use of a WC. 

We should bypass parse_repoarchdescr() or just invoke the wipe call of the build script before that. 

I am working on a fix. ETA tomorrow morning.

**Comment by nickbroon** [comment]
Fixed by #710
